doc/aursearch.md

```diff
@@ -0,0 +1,29 @@
+# aursearch
+
+## Name
+
+aursearch - Search AUR package names by PCRE pattern
+
+## Synopsis
+
+aursearch [+-Fbqrv} [--] ARGS...
+
+## Description
+
+Search the AUR for packages. The argument is a PCRE pattern to search the AUR with.
```

Comment:
Hmm... "search the AUR _with_" sounds a bit odd. Perhaps "to match against the AUR" or so.